Pages/Crime.cshtml

```diff
@@ -26,44 +27,56 @@
                     // display nothing if search doesnt have a return value
                 }
             }
+           
         }
+
     }
 </script>
 
-<input type="text" id="myInput" onkeyup="myFunction()" placeholder="Input a Zip Code">
+<input type="text" id="myInput" onkeyup="filterZipCode()" placeholder="Input a Zip Code">
 
 
 
 <div class="text-center">
     <h1 class="featurette-heading" style="color:crimson">Crime Information</h1>
-    <table id="crimetable" class="table table-bordered">
 
-        <tr>
-            <th>Date of Report</th>
-            <th>Type</th>
-            <th>Weapon Involvement</th>
-            <th>Location</th>
-            <th>Neighborhood</th>
-            <th>Address</th>
-            <th>Zip Code</th>
-        </tr>
+    @{
+        if (crimes.Length > 0)
```

Comment:
Logical change:-
check size of array and displaying message if array is empty

Reply:
Good.